app/javascript/redux/greetings/greetings.js

```diff
@@ -0,0 +1,31 @@
+const LOAD_GREETING = 'App/greetings/LOAD_GREETING';
+
+const URL = 'http://localhost:3000/v1/greetings';
+
+const initialState = {
+    id: 1000,
+    greeting: "Howdy CR"
+};
+
+export const loadGreeting = (payload) => ({
+    type: LOAD_GREETING,
+    payload
+});
+
+export const getGreetings = () => async (dispatch) => {
+    await fetch(URL)
+    .then((res) => res.json())
+    .then((greeting) => dispatch(loadGreeting(greeting)))
+    .catch((err) => console.log(err))
+}
```

Comment:
- [ ] The data is not fetched from the Rails API after clicking on the `Random greeting` button. The following error message appears on the console::
 
<img width="956" alt="image" src="https://user-images.githubusercontent.com/77190203/168452119-cfa1fffe-300f-4338-b1b5-803885343043.png">

- Please make sure that a random greeting from the Rails API is displayed after clicking on the `Random greeting` button.